README.md

```diff
@@ -144,6 +144,8 @@ Pipelines consist of a pipeline repository, launch parameters, and a Compute Env
 tw launch my_sleepy_pipeline
 ```
 
+> Add a `--wait=SUCCEEDED` if you want the command to wait until the pipeline execution is complete.
```

Comment:
Might be nice to have some mention of the different exit codes as well (and under which conditions they occur) 👍🏻

Reply:
I think the error code are only 0 (ok) and 1 (fail)

Reply:
Yup, but that's useful to know! 😉 

I was also slightly unclear on the nitty gritty. For example, if I do `--wait=FAILED` and the run finishes with `SUCCEEDED` will `tw` effectively hang, waiting forever? Or exit with `1`? Or exit with `0` because it was a successful run?

Reply:
```suggestion
Add a `--wait=SUCCEEDED` if you want the command to wait until the pipeline execution is complete.

When using `--wait`, `tw` can exit with one of two exit codes:

- `0`: When the run/compute-env reaches the desired state.
- `1`:  When the run/compute-env reaches a state that makes it impossible to reach the desired state.
```

Reply:
hang waiting forever (until you press Ctrl+C) :laughing:

Reply:
Ok, let me fix these corner cases. I'm going to return a 1 exit code, because I imagine that someone does a `--wait=FAILED` because he wants to test that the run fails (like a unit testing of a pipeline).